graylog2-web-interface/src/actions/users/UsersActions.js

```diff
@@ -40,13 +40,14 @@ export type PaginatedUsers = PaginatedList<UserOverview> & {
 
 export type ActionsType = {
   create: (user: UserCreate) => Promise<void>,
-  load: (username: string) => Promise<User>,
-  update: (username: string, request: any) => Promise<void>,
-  delete: (username: string) => Promise<void>,
-  changePassword: (username: string, request: ChangePasswordRequest) => Promise<void>,
-  createToken: (username: string, tokenName: string) => Promise<Token>,
-  loadTokens: (username: string) => Promise<Token[]>,
-  deleteToken: (username: string, tokenId: string, tokenName: string) => Promise<void>,
+  load: (userId: string) => Promise<User>,
+  loadByUsername: (username: string) => Promise<User>,
+  update: (userId: string, request: any) => Promise<void>,
+  delete: (username: string, fullName: string) => Promise<void>,
```

Comment:
Why don't we delete users by ID and still use the username for that?

Reply:
I didn't want to touch the delete code, because that also needs changes to the UserService.delete() which has these weird functionality that allows to remove multiple users with the same name..

Reply:
@mpfz0r How about creating a `UserService#deleteById` method and a `DELETE /users/id/<id>` endpoint so we can leave the other code untouched?

Reply:
@bernd that I can do, of course

Reply:
ok, done. @linuspahl there is also user deletebyId endpoint now